### ISSUE TITLE
Can't login after install

### ISSUE THREAD

**Issue by Zanatus** [lKinderBueno/Streamity-Xtream-IPTV-Web-player#15]
Install complete successfully but when I head to /login.php and insert my list user and password it says "Inserted info not valid."

**Comment by chrystal56** [comment]
Hi Zanatus, config.php 😉👍

**Comment by Zanatus** [comment]
config.php:
![image](https://user-images.githubusercontent.com/5156096/99826604-809bbc00-2b50-11eb-8951-1b0efcd585cd.png)

when I try to login:
![image](https://user-images.githubusercontent.com/5156096/99826675-990bd680-2b50-11eb-9ce5-88e85f90bb13.png)

I've also tried the URL version, it installs with no errors, but when I try to log in:
![image](https://user-images.githubusercontent.com/5156096/99826742-af199700-2b50-11eb-9ad9-cd6c051fa040.png)

**Comment by chrystal56** [comment]
In config.php

$Host= "url server iptv:port"

exemple

$Host= "http://vpro.viptvclub.com:25443"

login & password : iptv host

**Comment by Zanatus** [comment]
Yes, that's what I did, I just put example.com to take the screenshot, of course the host is not that :P but yes, I had "HTTP://mysite.com:8080"

**Comment by chrystal56** [comment]
Please give me u'r URL Host IPTV
No login no password just url

**Comment by Zanatus** [comment]
http://sinal.live:8080

**Comment by chrystal56** [comment]
ok look here & test ---> https://chrystal.me/

**Comment by Zanatus** [comment]
It worked.
What's different then? Ports? What?
Also, your login.php background is too heavy takes a lot to load the picture, on slow connections may take a while to charge

**Comment by chrystal56** [comment]
ok in u'r config.php ---> http://sinal.live:8080   =ko
                                ---->http://sinal.live:8080/ = ok

**Comment by chrystal56** [comment]
on the live page ssl is not supported url = http no https

**Comment by chrystal56** [comment]
it's OK for you ?

**Comment by Zanatus** [comment]
![image](https://user-images.githubusercontent.com/5156096/99845175-227cd200-2b6c-11eb-8c37-af266d64fb50.png)
This is my config.php, I haven't touched it.

**Comment by chrystal56** [comment]
$host="http://sinal.live:8080/";

**Comment by chrystal56** [comment]
It still does not work ?

**Comment by Zanatus** [comment]
nothing

**Comment by chrystal56** [comment]
then the problem is elsewhere i'm sorry

**Comment by Zanatus** [comment]
can you send me your player files, without your configs, so I can try out to see if it works? It's weird, I'm pretty sure I did it right, don't know why is not working at all

**Comment by chrystal56** [comment]
https://chrystal.me/1.1.Fixed.zip

**Comment by chrystal56** [comment]
I come back later in the evening. don't hesitate to keep me informed. I will try to find the solution with you

**Comment by Zanatus** [comment]
I've tried with the files you sent me but the same happens.

**Comment by chrystal56** [comment]
You use a free host ?,
 if this is the case the problem is certainly due to restriction on your host

**Comment by Zanatus** [comment]
no, it's not a free host.

**Comment by chrystal56** [comment]
before installation the permissions on the folders and files are in chmod 777 ?

After installation delete installer.php & finish.php 
& config.php chmod 644

**Comment by chrystal56** [comment]
I don't have a lot of information so I try to help you by going over the basics for the installation even if I suspect that you are proceeding in the right way it's always good to check

**Comment by Zanatus** [comment]
I've suspected about perms already but did not check, I will do that now, I will keep you informed, and thanks bro!! Is not everyday that somebody "wastes" his time to help some random guy online ;)

**Comment by chrystal56** [comment]
i really don't feel like wasting my time and i'm happy to be able to help you hoping it leads to a good result

**Comment by Zanatus** [comment]
I've tried the permissions but now it does not give me any error but it doesn't login at all as well..

Here, I've created an FTP user for you to test if you want. Let me know when you grab them so I can edit\delete this comment ;)

Login Name: tvtmea_0 || Password: jK6dfrYQ54H33Fgi
ftp: tv4me.club
test url: tv4me.club/play

**Comment by chrystal56** [comment]
Ok remove the FTP access I'll give you my email address so you don't leave any information lying around here

 You create a new FTP access and give me the information at admin@chrystal.me

**Comment by lionkingaw** [comment]
Hi, I am having the same problem as Zanatus. Installed, config is correct, permissions are correct. This is not a free host.

It does not let me login after install. Thanks

**Comment by daveondisc** [comment]
Hey guys, I have installed on a dedicated server and all seems fine. I can log in and it shows my channel count etc. but when I click on the live player, I just get a spinning circle and just sits there. Any help would be great....Thanks!

**Comment by lionkingaw** [comment]
> Hey guys, I have installed on a dedicated server and all seems fine. I can log in and it shows my channel count etc. but when I click on the live player, I just get a spinning circle and just sits there. Any help would be great....Thanks!

@daveondisc  What username and password did you use to login? Did you use an Xtream user/pass? I can't even login. Thanks

**Comment by daveondisc** [comment]
> 
> 
> > Hey guys, I have installed on a dedicated server and all seems fine. I can log in and it shows my channel count etc. but when I click on the live player, I just get a spinning circle and just sits there. Any help would be great....Thanks!
> 
> @daveondisc What username and password did you use to login? Did you use an Xtream user/pass? I can't even login. Thanks

Yes, log in with xtream user/pass but can not access live player?

**Comment by ccblog** [comment]
same issue can't login after fresh installation.

**Comment by ykose1982** [comment]
put the / at the end of the URL is gonno fix your problem

example: http://yoururl.com:80/

**Comment by MMELA15** [comment]
même problème ne peut pas se connecter après installation.

**Comment by MMELA15** [comment]
Install complete successfully but when I head to /login.php and insert my list user and password it says "Inserted info not valid."

**Comment by MMELA15** [comment]
> I've suspected about perms already but did not check, I will do that now, I will keep you informed, and thanks bro!! Is not everyday that somebody "wastes" his time to help some random guy online ;)

did you correct the problem ??
